build.gradle.kts

```diff
@@ -33,15 +33,25 @@ val test by tasks.getting(Test::class) {
 
 repositories {
     mavenCentral()
+    maven("https://dl.bintray.com/kotlin/exposed")
 }
 
 dependencies {
     compile("org.springframework.boot:spring-boot-starter-web")
     compile("org.springframework.boot:spring-boot-starter-data-jpa")
+    compile("org.springframework.boot:spring-boot-starter-actuator")
+    compile("org.springframework.boot:spring-boot-configuration-processor")
     compile("com.h2database:h2")
     compile("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
     compile("org.jetbrains.kotlin:kotlin-reflect")
-    compile("com.fasterxml.jackson.module:jackson-module-kotlin")
+    compile("com.fasterxml.jackson.module:jackson-module-kotlin:2.9.4.1")
+    // swagger
```

Comment:
Why?. this not apply to this task.

Reply:
The version is indicated in order that there should not be any questions and which one we use, and what documentation to look at. Specifically, this module has certain versions for certain versions of jackson.
And why do we need it at all?